README.md

```diff
@@ -1,59 +1,60 @@
+SSV uses an MPC threshold scheme with a consensus layer on top ([Istanbul BFT](https://arxiv.org/pdf/2002.03613.pdf)),
 that governs the network. \
-Its core strength is in its robustness and fault tolerance which leads the way for an open network of staking operators 
+Its core strength is in its robustness and fault tolerance which leads the way for an open network of staking operators
 to run validators in a decentralized and trustless way.
 
 ## Information
 
-### General SSV information (Semi technical read)
-* Article by [Blox](https://medium.com/bloxstaking/an-introduction-to-secret-shared-validators-ssv-for-ethereum-2-0-faf49efcabee)
+### 🔷 General SSV information (Semi technical read)
 
-### Technical iBFT and SSV read
-* [iBFT Paper](https://arxiv.org/pdf/2002.03613.pdf)
-    * [Fast sync for current instance](./ibft/sync/speedup/README.md)
-* [iBFT annotated paper (By Blox)](./ibft/IBFT.md)
-* [EIP650](https://github.com/ethereum/EIPs/issues/650)
-* [Security proof for n-t honest parties](https://notes.ethereum.org/DYU-NrRBTxS3X0fu_MidnA)
-* [MEV Research - Block proposer/ builder separation in SSV](https://hackmd.io/DHt98PC_S_60NbnW4Wgssg)
+- Article by [SSV](https://medium.com/bloxstaking/an-introduction-to-secret-shared-validators-ssv-for-ethereum-2-0-faf49efcabee)
+
+### 🔷 Technical iBFT and SSV read
+
+- [iBFT Paper](https://arxiv.org/pdf/2002.03613.pdf)
+- [iBFT annotated paper (By Blox)](./ibft/IBFT.md)
+- [EIP650](https://github.com/ethereum/EIPs/issues/650)
+- [Security proof for n-t honest parties](https://notes.ethereum.org/DYU-NrRBTxS3X0fu_MidnA)
+- [MEV Research - Block proposer/ builder separation in SSV](https://hackmd.io/DHt98PC_S_60NbnW4Wgssg)
+
+### 🔷 Audits
 
-### Audits
 - [Least Authority](/audits/Least%20Authority.pdf) (August 2023)
 
-## Getting Started
+## ✨ Getting Started
 
 The following documents contain instructions and information on how to get started:
-* [Operator Node Installation](https://docs.ssv.network/run-a-node/operator-node/installation)
-* [Developers' Guide](./docs/DEV_GUIDE.md)
-* [SSV API Docs](https://bloxapp.github.io/ssv/)
 
-## Contribution
+- [Operator Node Installation](https://docs.ssv.network/run-a-node/operator-node/installation)
+- [Developers' Guide](./docs/DEV_GUIDE.md)
+- [SSV API Docs](https://ssvlabs.github.io/ssv/)
+
+## <img src="https://github.com/Anmol-Baranwal/Cool-GIFs-For-GitHub/assets/74038190/fa83eeb9-f4e2-4d85-93f0-688af11babf8" width="35">&nbsp; Contribution
 
 Thank you for considering a contribution to the source code.
 
-In order to contribute to eth2-ssv, please fork, add your code, commit and send a pull request
+In order to contribute to `ssv`, please fork, add your code, commit and send a pull request
 for the maintainers to review and merge into the main code base.\
 If you wish to submit more complex changes though, please check up with the core devs first on [our discord](https://discord.gg/eDXSP9R)
```

Comment:
Is this Discord link working for you guys ? 

<img width="400" alt="image" src="https://github.com/user-attachments/assets/51f4d810-fc0c-430b-87ed-6dce9abc8070">

It should probably match the one we have below: https://discord.gg/ssvnetworkofficial ?

Reply:
same here, let's change it while we're touching this
<img width="1651" alt="image" src="https://github.com/user-attachments/assets/87fc868e-f9c2-4f21-bc1a-08466fcb1a30">

Reply:
thx, fixed